server/queries/queries.js

```diff
@@ -0,0 +1,39 @@
+const connection = require("../database/db_connection");
+
+const logMood = body => {
+  const { mood, date, time } = body;
+
+  return connection.query(
```

Comment:
## Let's add some error handling to ```logMood```
At the moment, the code in this function doesn't handle database errors. If something goes wrong when you're inserting data into the DB, it'll crash.

Since you're already (and wisely!) using promises in your ```index.js``` file, I'd suggest refactoring this db query to return a promise:

```
const logMood = body => {
  console.log("Logging mood...")
  const { mood, date, time } = body;
return new Promise((resolve, reject) => {
  connection.query(
    `INSERT INTO moods(user_id, mood, mood_date, mood_time )
     VALUES($1, $2, $3, $4)`,

    [1, mood, date, time]
  )
      .then(res => {
        console.log("Mood successfully logged to DB");
        resolve(true);
      })
.catch(err => reject(err))
});
}
```

By turning this into a promise with a ```.catch()``` statement, we've added error handling to your DB query. This will make it easier to debug when things go wrong :)